test/fixtures/transaction_builder.json

```diff
@@ -285,6 +285,28 @@
             "value": 10000
           }
         ]
+      },
+      {
+        "description": "Transaction w/ 1 pubKeyHash transaction input (Issue #644)",
+        "network": "testnet",
+        "txHex": "",
+        "inputs": [
+          {
+            "txHex": "0100000001f7e6430096cd2790bac115aaab22c0a50fb0a1794305302e1a399e81d8d354f4020000006a47304402205793a862d193264afc32713e2e14541e1ff9ebb647dd7e7e6a0051d0faa87de302205216653741ecbbed573ea2fc053209dd6980616701c27be5b958a159fc97f45a012103e877e7deb32d19250dcfe534ea82c99ad739800295cd5429a7f69e2896c36fcdfeffffff0340420f00000000001976a9145c7b8d623fba952d2387703d051d8e931a6aa0a188ac8bda2702000000001976a9145a0ef60784137d03e7868d063b05424f2f43799f88ac40420f00000000001976a9145c7b8d623fba952d2387703d051d8e931a6aa0a188ac2fcc0e00",
+            "vout": 0,
+            "signs": [
+              {
+                "keyPair": "cV6Fs4jspcMmUoPXyQSAVkVDfNVjL2Sn8HsBwpDvmKdhF5A9CkQW"
```

Comment:
@runn1ng this private key isn't right,  as specified in https://github.com/bitcoinjs/bitcoinjs-lib/issues/644#issuecomment-251851722